internal/api/core/route.go

```diff
@@ -106,5 +111,28 @@ func MustNewAPIRoute(basePath string, apiHandler any) *APIRoute {
 		},
 		RequestType:  requestType,
 		ResponseType: responseType,
+		Description:  "",
 	}
+
+	absPath := makeAbsLocalAPIPath(basePath) + ".go"
+	fset := token.NewFileSet()
+	node, parseErr := parser.ParseFile(fset, absPath, nil, parser.ParseComments)
+	if parseErr != nil {
+		log.Warn("Error while parsing file to get API description.", parseErr, log.NewAttr("endpoint", absPath))
+		return &apiRoute
+	}
+
+	for _, decl := range node.Decls {
+		function, ok := decl.(*ast.FuncDecl)
+		if !ok {
+			continue
+		}
+
+		if strings.Contains(function.Name.Name, "Handle") && function.Doc != nil {
```

Comment:
Could an anonymous function appear in the top-level declarations?
If so, would that have a nil name?

The `Name.Name` makes me a bit nervous about a panic.